### ISSUE TITLE
Workflow failed - km_mt_stress_tests

### ISSUE THREAD

**Issue by shpalani** [microsoft/ebpf-for-windows#3728]
### Describe the bug

Failed test case: **native_invoke_v4_v6_programs_restart_extension_test**
Pipeline: 
- https://github.com/microsoft/ebpf-for-windows/actions/runs/10066891733/
- https://github.com/microsoft/ebpf-for-windows/actions/runs/10080949547

```
[15:35:08] :: Starting test *** native_invoke_v4_v6_programs_restart_extension_test ***
[15:35:08] :: test threads per program    : 2
[15:35:08] :: test duration (in minutes)  : 5
[15:35:08] :: test verbose output         : false
[15:35:08] :: test extension restart      : false
[15:35:08] :: waiting on 2 test threads...
[15:35:08] :: _load_attach_program(1) FATAL ERROR: bpf_prog_attach(cgroup_count_connect6.sys) failed. program:count_tcp_connect6, errno:22
[15:35:08] :: ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[15:35:08] :: ebpf_stress_tests_km is a Catch2 v3.6.0 host application.
[15:35:08] :: Run with -? for options
[15:35:08] :: -------------------------------------------------------------------------------
[15:35:08] :: native_invoke_v4_v6_programs_restart_extension_test
[15:35:08] :: -------------------------------------------------------------------------------
[15:35:08] :: D:\a\ebpf-for-windows\ebpf-for-windows\tests\stress\km\stress_tests_km.cpp(1469)
[15:35:08] :: ...............................................................................
[15:35:09] :: D:\a\ebpf-for-windows\ebpf-for-windows\tests\stress\km\stress_tests_km.cpp(822): FAILED:
[15:35:09] ::   REQUIRE( result == 0 )
[15:35:09] :: with expansion:
[15:35:09] ::   -22 == 0
[15:35:09] :: 

[15:35:09] :: *** ERROR *** C:\eBPF\Run-Self-Hosted-Runner-Test.ps1: C:\eBPF\ebpf_stress_tests_km failed.
*** ERROR *** C:\eBPF\Run-Self-Hosted-Runner-Test.ps1: C:\eBPF\ebpf_stress_tests_km failed.
At C:\actions_runner\_work\ebpf-for-windows\ebpf-for-windows\x64\Debug\vm_run_tests.psm1:33 char:5
+     Invoke-Command -VMName $VMName -Credential $TestCredential -Scrip ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (*** ERROR *** C...ests_km failed.:String) [], RuntimeException
    + FullyQualifiedErrorId : *** ERROR *** C:\eBPF\Run-Self-Hosted-Runner-Test.ps1: C:\eBPF\ebpf_stress_tests_km fail 
   ed.
    + PSComputerName        : vm1_ws2019
 
Error: Process completed with exit code 1.
```

### OS information

Windows 10 and above

### Steps taken to reproduce bug

- Submit to 'Release Validation' pipeline with branch release/0.18

- .\ebpf_stress_tests_km.exe -vo=true -td=5 -tt=2 native_invoke_v4_v6_programs_restart_extension_test

### Expected behavior

native_invoke_v4_v6_programs_restart_extension_test should pass.

### Actual outcome

_load_attach_program(1) FATAL ERROR: bpf_prog_attach(cgroup_count_connect6.sys) failed. program:count_tcp_connect6, errno:22

**Comment by shankarseal** [comment]
@shpalani - please see if this is a dupe of auto generated bug, and if so close. Otherwise, please work with me to investigate this further and provide a fix by 2408.

**Comment by shpalani** [comment]
Duplicate of #3607

**Comment by shpalani** [comment]
Duplicate of #3607
